### PR TITLE
First approach to github projet guidelines documentation

### PR DESCRIPTION
**Pull request by vctr90**
Hi @tecnoyucas/yucazos 

This PR seeks to solve #10 Issue about GitHub projects guidelines.

Please review this pull request and also let's discuss about this document. You might want to suggest some changes or new flows, I'm all ears :smile: 

Thanks.

### CODE REVIEW

File: documentation/github-project-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+﻿# Guidelines for our GitHub projects
+
+In this place we will have the documentation for how to handle issues
+among projects of **TecnoYucas**.
+
+## General notes
+
+### Some rules first about the project
```
Comment (aitbw) [inline documentation/github-project-guidelines.md]: I think it would be better if this was something like `General rules for projects`

File: documentation/github-project-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+4. All issues should have a deliverable product (a document or a
+merged branch, this would help on issue completion agreement).
+
+### Some recommendations
+
+- Keep language as complete and concise as possible.
+- Use Todo lists (this can help to better define the scope of an issue
+  and the state of it).
+- Please try not to create huge issues, make them as short and clear
+  scoped as possible. If you need some sort of complex issue, please
```
Comment (aitbw) [inline documentation/github-project-guidelines.md]: I think `scoped` slipped in here :sweat_smile:

File: documentation/github-project-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  brake it down into smaller pieces.
```
Comment (aitbw) [inline documentation/github-project-guidelines.md]: Typo: Change it for `break`

**Comment by aitbw** [comment]
One final change, @vctr90: Could you please move this file to [this folder](https://github.com/tecnoyucas/meta/tree/master/do-not-remove)? There's an empty file there especially for this document

Other than that, I agree with this draft :smile:

File: documentation/github-project-guidelines.md (hunk 2 above)
Comment (vctr90) [reply documentation/github-project-guidelines.md]: @aitbw Why do you say it is slipped?

File: documentation/github-project-guidelines.md (hunk 3 above)
Comment (vctr90) [reply documentation/github-project-guidelines.md]: Sure, thanks

**Comment by vctr90** [comment]
@aitbw Please review again. Thanks.

File: documentation/github-project-guidelines.md (hunk 2 above)
Comment (aitbw) [reply documentation/github-project-guidelines.md]: Never heard of `short and clear scoped` before

**Comment by aitbw** [comment]
Please, include this document [here](https://github.com/tecnoyucas/meta/blob/master/do-not-remove/projects_guidelines.md) or delete that file to avoid duplicates :relieved:

**Comment by vctr90** [comment]
Done @aitbw

File: do-not-remove/projects_guidelines.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+## General notes
+
+### Some rules first about the project
+
+1. Every repo must have a automatic Kanban-like GitHub project
+2. Every single pull request MUST be associated with a GitHub issue
+3. Whenever you close a pull request or comment on an issue, please
+make sure to agree on issue completion.
+4. All issues should have a deliverable product (a document or a
```
Comment (sadasant) [inline do-not-remove/projects_guidelines.md]: This is not consistent with our current `DO NOT REMOVE` issues. Let’s rephrase this to be relative to the Publication-N repos and not the meta repo.

File: do-not-remove/projects_guidelines.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  and the state of it).
+- Please try not to create huge issues, make them as short and clear
+  scoped as possible. If you need some sort of complex issue, please
+  break it down into smaller pieces.
+
+### The workflow
+
+1. When an issue is newly created, it will directly go into `Inbox
+(Not approved yet)` column
+2. After an issue is approved (by at least **two** members of the
```
Comment (sadasant) [inline do-not-remove/projects_guidelines.md]: we can automate this if we use waffle instead of github projects and treat the boards to use labels, so an issue can go from Inbox to `To  do (Approved)` as soon as someone assigns him/herself to the issue.

**Comment by stefanmaric** [comment]
> Please, include this document here or delete that file to avoid duplicates

I think this pretty much fits what `CONTRIBUTING.md` is for and should be placed into that file as such for Github to catch it: https://help.github.com/articles/setting-guidelines-for-repository-contributors/

This is a very widespread practice, to not call it standard, that helps newcomers discover valuable information about the project.

File: do-not-remove/projects_guidelines.md (hunk 4 above)
Comment (vctr90) [reply do-not-remove/projects_guidelines.md]: @sadasant When I say "deliverable product", I mean, a generic way to give a "tangible" solution for an issue, the issue could be related or not to a publication. For example, this is not a document considered as TecnoYucas publication, therefore, it is just a guideline document, a "deliverable product" of this issue. What I can do is to put a note that states that a "Deliverable product" could also be a Publication-N.

I am guessing TecnoYucas will be wider than just publications. We will be doing things not related to publications. For example: A merged feature, is a "deliverable product" of an issue but as you know, it does not have necessarily be a publication-related issue.

File: do-not-remove/projects_guidelines.md (hunk 5 above)
Comment (vctr90) [reply do-not-remove/projects_guidelines.md]: Cool, I can investigate and set a project in Waffle, what do you think @sadasant ?

Comment (stefanmaric) [reply do-not-remove/projects_guidelines.md]: I haven't used Waffle, but Github Projects have something similar: https://help.github.com/articles/configuring-automation-for-project-boards/

File: do-not-remove/projects_guidelines.md (hunk 4 above)
Comment (sadasant) [reply do-not-remove/projects_guidelines.md]: Yes! I understand that, I think. However, we have created issues that have no deliverables, which are our DO NOT REMOVE issues. We use them as a place to communicate. These issues were created with the goal of not removing them for now. What are your thoughts around that?

File: do-not-remove/projects_guidelines.md (hunk 5 above)
Comment (sadasant) [reply do-not-remove/projects_guidelines.md]: Yes! Well, essentially: all of the columns should be automated based on tags, once we do that, I'll be comfortable with having rules around this. None of this process should be manual (except for some of the issue assignment at least)
